dispatches/models/nuclear_case/Hydrogen_Turbine/hturbine_ideal_vap.py

```diff
@@ -0,0 +1,196 @@
+##############################################################################
```

Comment:
Minor comment, but do we want to put this in a separate "properties" folder somewhere?